include/wx/utils.h

```diff
@@ -601,8 +601,8 @@ WXDLLIMPEXP_BASE bool wxGetDiskSpace(const wxString& path,
                                      wxDiskspaceSize_t *pTotal = NULL,
                                      wxDiskspaceSize_t *pFree = NULL);
 
-
-
+// wxQsort can be initially declared in wx/vector.h
```

Comment:
Only if wxUSE_STD_CONTAINERS == 0. Would probably be better to use a specific preprocessor symbol to indicate whether wxQsort is declared, something like WXQSORT_DECLARED perhaps.